lib/Home.dart

```diff
@@ -42,14 +42,16 @@ class Home extends StatefulWidget {
 
 class _HomeState extends State<Home> {
   int _selectedIndex = 0;
-
+  List<UpcomingRide> rides;
   String _name;
 
   @override
   void initState() {
     super.initState();
     // Fetch name of user
     _name = "Chris";
+    // TODO: fetch info about rides
+    rides = new List();
```

Comment:
If this is non-empty, the number of rides that should appear should be rides.length, with the first ride being that top card, and the rest being in the schedule. You may choose to address this here or in your refactoring.

Reply:
Yes, I'll be adding this in the refactoring, thanks!